sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -305,7 +305,7 @@ export class ServiceBusManagementClient extends ServiceClient {
    * @throws `RestError` with code `ServiceError` when receiving unrecognized HTTP status or for a scenarios such as
    * bad requests or requests resulting in conflicting operation on the server,
    * @throws `RestError` with code that is a value from the standard set of HTTP status codes as documented at
-   * https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=netframework-4.8
+   * https://docs.microsoft.com/dotnet/api/system.net.httpstatuscode?view=netframework-4.8
```

Comment:
[Unrelated to the PR]
.NET Link?

Reply:
I believe the error code is coming from the service, and the JS SDK does not add any translation or edits. So, while it is preferred to have a language agnostic source where this is documented, I wonder if the fact that service is using .Net brings in value to have this documented like so. Worth digging into.

Reply:
Perhaps https://docs.microsoft.com/en-us/rest/api/storageservices/common-rest-api-error-codes is better?

Reply:
Looks more relevant I'd say!

Reply:
This is the link that is referenced in the docs such as `create-queue` and `update-queue`.

Reply:
Ah, but that link is under "storageservices", I'll send another PR to add some generic text here rather than using the link

Reply:
See #10700